src/button/validate.js

```diff
@@ -127,6 +128,29 @@ export function validateButtonStyle(style : Object = {}) {
             throw new Error(`style.${ BUTTON_STYLE_OPTIONS.TAGLINE } is not allowed for ${ BUTTON_LAYOUT.VERTICAL } layout - got ${ tagline }`);
         }
     }
+
+    validateRegionSpecificButton(style, props.locale);
+
+}
+
+export function validateRegionSpecificButton(style : Object = {}, locale : string) {
+
+    let country = locale.split('_')[1];
+
+    let isInstallmentAllowedCountry = ALLOWED_INSTALLMENT_COUNTRIES.indexOf(country) !== -1;
+
+    if ( !isInstallmentAllowedCountry && style.installmentperiod ) {
```

Comment:
Let's use `style[BUTTON_STYLE_OPTIONS.INSTALLMENTPERIOD]` then any future name changes only need to happen in one place.

Reply:
👍